samples/hono/src/handlers/createPets.ts

```diff
@@ -0,0 +1,11 @@
+import { createFactory } from 'hono/factory';
+import { zValidator } from '@hono/zod-validator';
+import { CreatePetsContext } from '../petstore.context';
+import { createPetsBody } from '../petstore.zod';
+
+const factory = createFactory();
+
+export const createPetsHandlers = factory.createHandlers(
+  zValidator('json', createPetsBody),
+  (c: CreatePetsContext) => {},
```

Comment:
@anymaniax 

I wasn't able to comment properly...

Since there is no longer a default implementation, c is now unused. Depending on the user's formatter settings, a warning will occur. Is this an intended change?

Reply:
It’s normally always need to be implemented so shouldn’t be an issue